src/client/app/containers/AcctInputContainer.js

```diff
@@ -1,56 +1,60 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
-import Input from '../components/Input'
+
 import { validateInput } from '../services/'
 import { socket } from '../store/socket'
+
+import Input from '../components/Input'
+
 // action creators
 import { 
     submitAcct,
     validateClient
 } from '../store/reducers/acctInput'
+
 // action keys
 import {
     SUBMIT_ACCT_INPUT,
     RESPONSE_VALIDATE_CLIENT
 } from '../store/actions/'
 
-class AcctInput extends Component {
-    constructor(props) {
-        super(props),
-        this.state = {
-            inputValue: ''
-        }
-    }
-
-    handleInputChange(e) {
-        this.setState({ inputValue: validateInput(e.target.value) })
-    }
-
-    handleInputSubmit() {
-        this.setState( {inputValue: ""} )
-        this.props.submit(this.state.inputValue)
+class AcctInputContainer extends Component {
+    state = {
+        inputValue: ''
     }
```

Comment:
I've been ditching `constructor(props){ super(props) }` for this, idk just seems like extra lines especially since you're binding `this` to methods inside render